app/vite.config.ts

```diff
@@ -31,6 +31,7 @@ export default defineConfig({
   base: '/',
   build: {
     outDir: 'build',
+    sourcemap: 'inline'
```

Comment:
Do you want to put that in here -  would prefer it in a separate PR.

Reply:
Reverted- will put in separate PR